core/client/hammer/hammer.go

```diff
@@ -76,125 +93,159 @@ func New(ctx context.Context, dial DialFunc, callOptions CallOptions,
 		timeout:     timeout,
 		ktCli:       ktCli,
 		appID:       fmt.Sprintf("hammer_%v", time.Now().Format("2006-01-02/15:04:05")),
-		config:      config,
+		domain:      domain,
 
 		signers:        []*tink.KeysetHandle{keyset},
 		authorizedKeys: authorizedKeys.Keyset(),
 	}, nil
 }
 
 // Run runs operationCount with up to maxWorkers over ramp duration.
-func (h *Hammer) Run(ctx context.Context, operationCount, maxWorkers int, ramp time.Duration) {
-	jobs := gen(ctx, operationCount)
-	outputs := h.launchWorkers(ctx, jobs, maxWorkers, ramp)
-	h.printStats(1*time.Second, outputs)
-}
-
-// launchWorkers gradually adds workers to the worker pool over ramp up to workers.
-func (h *Hammer) launchWorkers(ctx context.Context, jobs <-chan int, workers int, ramp time.Duration) <-chan error {
-	outputs := make(chan error)
-
-	go func() {
-		rampTicker := time.NewTicker(ramp / time.Duration(workers))
-		var wg sync.WaitGroup
-		for h.workers < workers {
-			h.workers++
-			wg.Add(1)
-			go func() {
-				defer wg.Done()
-				h.worker(ctx, jobs, outputs)
-			}()
-			<-rampTicker.C
-		}
-		rampTicker.Stop()
+// workers should roughly be (goal QPS / timeout seconds)
```

Comment:
Comment out of date (no ramping any more)?

Reply:
Yep. Removed.